lib/resources/package.rb

```diff
@@ -194,15 +194,23 @@ class Brew < PkgManagement
     def info(package_name)
       brew_path = inspec.command('brew').exist? ? 'brew' : '/usr/local/bin/brew'
       cmd = inspec.command("#{brew_path} info --json=v1 #{package_name}")
+
+      # If no available formula exists, then `brew` will exit non-zero
       return {} if cmd.exit_status.to_i != 0
-      # parse data
+
       pkg = JSON.parse(cmd.stdout)[0]
-      {
-        name: pkg['name'],
-        installed: true,
-        version: pkg['installed'][0]['version'],
-        type: 'brew',
-      }
+      # If package exists but is not installed, then `brew` output will not
+      # contain `pkg['installed'][0]['version']
+      unless pkg.dig('installed', 0, 'version')
```

Comment:
`unless`...`else` is really frowned-upon in Rubyland.

Let's just make this:

```ruby
return {} unless pkg.dig('installed', 0, 'version')

# the package is installed, so we build up our pkg info hash
{
  name: ...
}
```

Reply:
I like that much more. Implemented in latest commit.

Thanks